back-end/src/api/app_api_v1.py

```
@@ -98,6 +99,44 @@ def get_application(app_id):
         return {"message": "No application found with given ID."}, 400
 
 
+@app_api_v1.route('/lcgen', methods=['POST'])
+@login_required
+def langcode_application():
+    data = request.get_json()
+    create_with_inst = data.get('create_with_inst', None)
+
+    json_data = {
+        "agent_inst": create_with_inst
+    }
+    response = requests.post(
+        'https://lang-py-522564686dd7.herokuapp.com/anchoring',
+        json = json_data
+    )
+    if response.status_code != 200:
+        raise SystemError(
+            f'Failed to get valid response from server: {response.status_code}')
```

Better to propagate the error to API users, something like:

return Response(".......", status=500)